### PR TITLE
Printf %d, fix Resolution rendering and GUI working in any size of window - Part 1

### PR DESCRIPTION
**Pull request by ElectroBoy404NotFound**
Add %d to printf, fix resolution not rendering properly and GUI elements were not working on window sizes that are not coded in. I changed the fixed values to calculate them when needed.

### CODE REVIEW

**Comment by ElectroBoy404NotFound** [comment]
-280 lines and +74 lines... My biggest pull request yet.

**Comment by Andrej123456789** [comment]
When I close TaskBar, `START` word is not printed

**Comment by ElectroBoy404NotFound** [comment]
> When I close TaskBar, `START` word is not printed

What do you mean by "When i close TaskBar"?

**Comment by Andrej123456789** [comment]
When I close StartMenu

**Comment by ElectroBoy404NotFound** [comment]
> When I close StartMenu

Working for me

**Comment by Andrej123456789** [comment]
Send screenshot

**Comment by ElectroBoy404NotFound** [comment]
> Send screenshot

Where is it?

**Comment by Andrej123456789** [comment]
> When I close StartMenu, `START` word is not printed

File: ringOS-Beta17/src/Window/Window.cpp [hunk 1]
```diff
@@ -183,14 +129,8 @@ namespace GUI
                 GlobalRenderer->BMPPicture();
                 GlobalRenderer->TaskBar();
                 break;
```
Comment (Andrej123456789) [inline ringOS-Beta17/src/Window/Window.cpp:135]: Add 
```
GlobalRenderer->CursorPosition = {0, (long) ResoHeight - 80};
printf("START")
```

**Review by Andrej123456789 — CHANGES_REQUESTED** [review]

File: ringOS-Beta17/src/Window/Window.cpp (hunk 1 above)
Comment (ElectroBoy404NotFound) [reply ringOS-Beta17/src/Window/Window.cpp:135]: ok

Comment (Andrej123456789) [reply ringOS-Beta17/src/Window/Window.cpp:135]: Add red colour

**Comment by Andrej123456789** [comment]
Things you need to do:

- [ ] SubMenu
- [ ] Close window

**Comment by ElectroBoy404NotFound** [comment]
> Things you need to do:
> 
>     * [ ]  SubMenu
> 
>     * [ ]  Close window

working on

**Comment by Andrej123456789** [comment]
Add red colour

**Comment by Andrej123456789** [comment]
I will added it

**Comment by Andrej123456789** [comment]
I will merge this because I will do part 2 because it require some other things

**Comment by Andrej123456789** [comment]
Thank you for Pull Request :)

**Pull request merged**